aunt-leahs-app/src/pages/VolunteerCheckInPage.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+
+import TextInput from '../components/TextInput'
+import SelectBox from '../components/SelectBox'
+import CustomButton from '../components/CustomButton'
+import Header from '../components/Header'
+
+import { setDuration } from '../redux/volunteer/volunteerAction';
+import { setCurrentPage } from '../redux/page/pageAction';
+
+function VolunteerCheckInPage({ history, setDuration, setCurrentPage }) {
+
+	let selectedDurration = null;
+	function selectDurration(durration) {
+		selectedDurration = durration
+	}
+
+	const setDurationIfSelected = () => {
+		if (selectedDurration) {
+			setDuration({ selectedDurration });
+			alert('You have selected' + selectedDurration);
+		}
+		else {
+			alert('You have not selected a duration!');
+		}
+	}
+	const moment = require('moment');
```

Comment:
Use the new ES6 feature here. You add to the top `import { moment } from 'moment';` instead of using `require`